data/sql/updates/pending_db_world/rev_1632568180601547400.sql

```diff
@@ -0,0 +1,8 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1632568180601547400');
+
+UPDATE `smart_scripts` SET `action_param3`='990000' WHERE  `entryorguid`=179666 AND `source_type`=1 AND `id`=0 AND `link`=0;
```

Comment:
action_param3 is an unsigned int, not a string
```suggestion
UPDATE `smart_scripts` SET `action_param3`=990000 WHERE  `entryorguid`=179666 AND `source_type`=1 AND `id`=0 AND `link`=0;
```